pulsar-broker/src/test/java/org/apache/pulsar/client/api/BrokerServiceLookupTest.java

```diff
@@ -402,8 +402,6 @@ public void testPartitionTopicLookup() throws Exception {
         producer.close();
         consumer.unsubscribe();
         consumer.close();
-        admin.topics().deletePartitionedTopic(topicName.toString());
```

Comment:
Could we use `deletePartitionedTopicAsync`?
For if we delete this line and the test execute with retry, could it occur the 'the topic existed' exception?

Reply:
I found no retry code.